test/dotnet-test.Tests/project.json

```diff
@@ -19,7 +19,7 @@
     "System.Net.Sockets": "4.1.0-rc2-24015",
     "System.Runtime.Serialization.Primitives": "4.1.1-rc2-24015",
     "xunit": "2.1.0",
-    "dotnet-test-xunit": "1.0.0-dev-140469-38"
+    "dotnet-test-xunit": "1.0.0-rc2-152011-55"
```

Comment:
All our tests should be targeting the same `dotnet-test-xunit`.  If you are going to move one up, I think you should move them all up at the same time.